images/fcos-bastion-image/root/usr/share/bastion_services/bind9/named.conf

```diff
@@ -0,0 +1,60 @@
+//
```

Comment:
@jadhaj these are not the options we currently have in our bastion.

Either use the information in ansible playbook in the jira or login the openshift-qe-bastion and grab the information from the /etc/bind folder in the docker container running bind9.

I'll reach offline to coordinate about that